docs/notebooks/analysis/nullpoint.ipynb

```diff
@@ -40,12 +40,12 @@
    "source": [
     "## Contents:\n",
     "\n",
-    "1. [How Null Point Finder works](#How-Null-Point-Finder-works)\n",
+    "1. [How the null point finder works](#How-the-null-point-finder-works)\n",
```

Comment:
There now seems to be a check that references in example notebooks actually work.  Cool!